DEV-2459: Review and Improvements (#643)
cloudposse/docs: docs/layers/accounts/deploy-accounts.mdx

    Before performing the "Deploying Accounts" step, the root account needs to be configured as an AWS Organization.

    This process also enables [AWS RAM for Organizations](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_enable-ram.html) via a CLI command, which is required for connecting the Organization.
  </Step>

  <Step>
    ## <StepNumber/> Raise Account Limits

    To deploy all accounts, we need to request an increase of the Account Quota from AWS support, which requires an AWS Organization to be created first.

    From the `root` account (not `SuperAdmin`), increase the [account quota to 20+](https://us-east-1.console.aws.amazon.com/servicequotas/home/services/organizations/quotas) for the Cloud Posse reference architecture, or more depending on your business use-case

  </Step>

  <Step>
    ## <StepNumber/> Deploying Accounts

>     <Note title="important">

Reviewer: ```suggestion
    <Note title="Important">
```

Author: I've made all of these all lowercase. Do you want me to change them all to first letter caps?

      With the addition of support for dynamic Terraform roles, our `baseline` cold start refarch layer now depends
      on/requires that we have `aws-teams` and `aws-team-roles` stacks configured. This is because `account-map` uses those
      stacks to determine which IAM role to assume when performing Terraform in the account, and almost every other component
      uses `account-map` (indirectly) to chose the role to assume. However, these components do _not_ need to be deployed yet.
    </Note>

>     Again verify the "account" configuration in `stacks/catalog/account.yaml`. **This is the point of no return!**
> 

Reviewer: I don't think it's clear to a newbie what we mean, when we say the point of no return in the context of account configuration. Sounds negative. 

```suggestion
    Again verify the "account" configuration in `stacks/catalog/account.yaml`. **This is the point of no return!**

```

Author: rephrased. Less urgent / negative

    These deployments will create all AWS member accounts and store relevant account metadata as "mappings" in the Terraform
    outputs of the `account-map` component. Rather than querying this `account` component each time we need an Account ID or
    role, we provision a static component `account-map`.

>     <Note title="IMPORTANT">

Reviewer: I believe we use CSS to make it uppercase. 

```suggestion
    <Note title="Important">
```

Author: replaced

  <Step>
    ### <StepNumber/> ClickOps to Complete Account Setup

    <Steps>
      1. For each new account:
         <Steps>
         1. Perform a password reset by attempting to log in to the AWS console as a "root user" using that account's email address
         1. Click the "Forgot password?" link
         1. You will receive a password reset link via email, which should be forwarded to the shared Slack channel for automated messages. Click the link
         1. Enter a new password

            <Note title="tip">

>              Use 1Password or Random.org to create a password 26-38 characters long, including at least 3 of each class of character: lower case, uppercase, digit, and symbol. You may need to manually combine or add to the generated password to ensure 3 symbols and digits are present.

Reviewer: Manually combine what? Are these requirements by AWS that get validated?

Author: I dont know why that sentence is included. I removed it

            </Note>

         1. Save the email address and generated password as web login credentials in 1Password

>          1. Finally, save the account number in a separate field

Reviewer: Not sure if this should be a separate step, or just part of the previous step.

```suggestion
         1. Finally, record the account number in a separate field of the 1Password item, and save it.
```

         </Steps>

      1. Log in using the new password, choose "My Security Credentials" from the account dropdown menu and set up Multi-Factor Authentication (MFA) to use a Virtual MFA device.

>       1. Save the MFA TOTP key in 1Password by using 1Password's TOTP field and built-in screen scanner, and save the Virtual MFA ARN

Reviewer: ```suggestion
      1. Save the MFA TOTP key in 1Password with 1Password's TOTP field by using the built-in screen scanner to save the Virtual MFA ARN
```

Author: Cleaned up this whole section

      1. Enable any necessary optional regions

>       1. Optional, but highly recommended - unsubscribe the account's email address from all marketing emails

Reviewer: How to unsubscribe?

Author: added
